NEWS.md

```diff
@@ -2,6 +2,10 @@
 This is a log of major changes in Gadfly between releases. It is not exhaustive.
 Each release typically has a number of minor bug fixes beyond what is listed here.
 
+# Version next
+
+ * Support AbstractVectors everywhere (e.g. `Guide.xticks(ticks=1:10)`) (#1293)
+
```

Comment:
next version will be 1.1.0, so please move this below.  otherwise, looks great!  thanks again for this.  will merge as soon as this change is made.

Reply:
Done!